index.html

```diff
@@ -13,11 +13,127 @@
   <link rel="stylesheet" href="css/reset.css">
   <link rel="stylesheet" href="css/layout.css">
   <link rel="stylesheet" href="css/styles.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link
+    href="https://fonts.googleapis.com/css2?family=DM+Sans:wght@500&family=Open+Sans&family=Proza+Libre:wght@400;600&display=swap"
+    rel="stylesheet">
 </head>
 <body>
   <header>
+
+    <div class="container">
+      <div class="row header-height">
+        <div class="column-half height-fit-content align-self-center">
+          <p class="logo">Code Journal</p>
+          <p class="entries-nav">Entries</p>
```

Comment:
This should be an `a` element, it is meant to swap the view to the entries view. You also don't seem to have any JavaScript for this link right now to handle that view swapping.

Reply:
![image](https://user-images.githubusercontent.com/99709530/162097183-7109770c-fc0b-4217-9c35-2f649c6c6a87.png)

Reply:
![image](https://user-images.githubusercontent.com/99709530/162273204-5bfd689f-6db3-4c24-91cb-038f802a6e0e.png)